infer_mapping_and_eval.py

```
@@ -6,112 +6,112 @@
 
 P_chunks = []
-for i in tqdm(range(0, len(ds), BATCH)):
-    P_chunks.append(predict_probs(ds[i:i+BATCH]["text"]))
-P = np.concatenate(P_chunks, axis=0)  # (N, num_labels)
+for i in tqdm(range(0, lends, BATCH)):
+    P_chunks.append(predict_probsds[i:i+BATCH]["text"])
+P = np.concatenateP_chunks, axis=0  # N, num_labels
 
 # Correlation matrix C between model heads and dataset labels
-def safe_corr(a, b):
+def safe_corra, b:
```

Ruff (0.12.2)

SyntaxError: Expected '(', found ','

---

SyntaxError: Expected an expression

---

SyntaxError: Expected ',', found indent

---

SyntaxError: Parameter without a default cannot follow a parameter with a default

---

SyntaxError: Expected ',', found '.'

---

SyntaxError: Parameter without a default cannot follow a parameter with a default

---

SyntaxError: Expected ')', found '('

---

Multiple statements on one line (colon)

(E701)